app/models/spotlight/resource.rb

```diff
@@ -30,10 +30,10 @@ def self.class_for_resource r
     end
 
     def to_solr
-      {
-        spotlight_resource_id_ssim: "#{(type.tableize if type) || self.class.to_s.tableize }:#{id}",
-        spotlight_resource_url_ssim: url
-      }
+      exhibit.solr_data.merge({
```

Comment:
For some reason, I thought you were calling `super` there. I'll take a look at it tomorrow and try to reconcile the two.